Formula/kubernetes-cli.rb

```diff
@@ -2,11 +2,11 @@ class KubernetesCli < Formula
   desc "Kubernetes command-line interface"
   homepage "https://kubernetes.io/"
   on_arm do
-    url "https://dl.k8s.io/release/v1.24.13/bin/darwin/arm64/kubectl"
+    url "https://dl.k8s.io/release/v1.26.10/bin/darwin/arm64/kubectl"
     sha256 "a0b30642de4e0c290e08f1895d7a0951a004f23d107bef86a6460c5f5b516585"
```

Comment:
We need to update the sha256 value to match the new binary. For each of these binaries:
1. `wget <url>`
2. `sha256sum <downloaded file> | pbcopy>
3. Paste sha256 sum here